### PR TITLE
Adding a short description of the REST API.

### PR DESCRIPTION
**Pull request by joivo**
As in the title.

### CODE REVIEW

**Review by thiagomanel — CHANGES_REQUESTED** [review]
I missed the capability of retrieving the size of the queue (the number of waiting jobs)

**Comment by joivo** [comment]
> I missed the capability of retrieving the size of the queue (the number of waiting jobs)

Maybe solved in 2cf92776395f3346f88ae8ab9fb8200640c11956.

**Review by wesleymonte — APPROVED** [review]

File: docs/api/README.md [hunk 1]
```diff
@@ -0,0 +1,302 @@
...
+POST /queues
+```
+
+**Request example**
+
+*Body*
+```json
+{
+   "name":"some_awesome_name",
+   "worker_nodes": [
```
Comment (thiagomanel) [inline docs/api/README.md]: how about renaming `worker_nodes` to `worker_pool`. Also, rename `worker_pool` to `pool_size`

Comment (thiagomanel) [reply docs/api/README.md]: `worker_pools`

File: docs/api/README.md [hunk 2]
```diff
@@ -0,0 +1,302 @@
...
+```
+
+**Response example**
+```json
+[
+    {
+      "id": "some_unique_id",
+      "name": "awesome_name",
+      "waiting_jobs": 2,
+      "worker_nodes": 5,
```
Comment (thiagomanel) [inline docs/api/README.md]: replace worker_nodes by worker_pools?

Comment (thiagomanel) [reply docs/api/README.md]: pools_size instead of worker_pool

Comment (thiagomanel) [reply docs/api/README.md]: replace in other places too

Comment (thiagomanel) [reply docs/api/README.md]: @emanueljoivo add the possible STATES in the README

File: docs/api/README.md (hunk 1 above)
Comment (joivo) [reply docs/api/README.md]: Ok.

File: docs/api/README.md (hunk 2 above)
Comment (joivo) [reply docs/api/README.md]: New version 1f3a9d353cca5e96b9bca4bfc39b0e7fda66ebd9.

**Review by thiagomanel — CHANGES_REQUESTED** [review]
I'm ready to accept. I still have only one small request: please, indicate remove the line 10 (we decide to ignore /workers, right).

Also, please indicate, in the first lines, that everything related to workers are pools are deprecated

**Comment by joivo** [comment]
> I'm ready to accept. I still have only one small request: please, indicate remove the line 10 (we decide to ignore /workers, right).
> 
> Also, please indicate, in the first lines, that everything related to workers are pools are deprecated

Done in ee23dbd8a2e79217d9c6860acf7ea2a605a02011.

**Review by thiagomanel — APPROVED** [review]
thanks, @emanueljoivo

**Pull request merged**